apps/logements/src/transformation/domain/service/studapart/convertir.domain-service.ts

```diff
@@ -62,7 +63,7 @@ export class Convertir {
 			servicesInclus: this.depuisOptionLogements(studapartLogement.options),
 			servicesOptionnels: [],
 			prixHT: 0,
-			prix: Number(studapartLogement.min_rent_with_charges),
+			prix: studapartLogement.min_rent_with_charges ? Number(studapartLogement.min_rent_with_charges) : Number(studapartLogement.full_property_rent_with_charges),
```

Comment:
Ça devait pas être l'inverse ? 
```suggestion
			prix: studapartLogement.full_property_rent_with_charges ? Number(studapartLogement.full_property_rent_with_charges) : Number(studapartLogement.min_rent_with_charges),
```